conf/checksums

```diff
@@ -4,22 +4,15 @@ fluo:1.2.0:037f89cd2bfdaf76a1368256c52de46d6b9a85c9c1bfc776ec4447d02c813fb2
 fluo_yarn:1.0.0:c6220d35cf23127272f3b5638c44586504dc17a46f5beecdfee5027b5ff874b0
 hadoop:3.3.1:2fd0bf74852c797dc864f373ec82ffaa1e98706b309b30d1effa91ac399b477e1accc1ee74d4ccbb1db7da1c5c541b72e4a834f131a99f2814b030fbd043df66
 hadoop:3.2.2:054753301927d31a69b80be3e754fd330312f0b1047bcfa4ab978cdce18319ed912983e6022744d8f0c8765b98c87256eb1c3017979db1341d583d2cee22d029
-hadoop:3.1.4:f7eff67e948e21e23d7069dc169c98fcce757f59473e1e9a0e27d69a23df88840a9776089eeae5a8e1e496753f811868fe65638b04e3388c197931cd44d86f63
 hadoop:3.0.3:db96e2c0d0d5352d8984892dfac4e27c0e682d98a497b7e04ee97c3e2019277a
 hadoop:2.10.1:2460E02CD1F80DFED7A8981BBC934C095C0A341435118BEC781FD835EC2EBDC5543A03D92D24F2DDEEBDFE1C2C460065BA1D394ED9A73CBB2020B40A8D8B5E07
-hadoop:2.9.2:3d2023c46b1156c1b102461ad08cbc17c8cc53004eae95dab40a1f659839f28a
-hadoop:2.8.5:F9C726DF693CE2DAA4107886F603270D66E7257F77A92C9886502D6CD4A884A4
-hadoop:2.7.6:f2327ea93f4bc5a5d7150dee8e0ede196d3a77ff8526a7dd05a48a09aae25669
-hadoop:2.6.5:001ad18d4b6d0fe542b15ddadba2d092bc97df1c4d2d797381c8d12887691898
```

Comment:
These should be kept. We still do release testing against all these versions for Accumulo.